layers.go

```diff
@@ -178,13 +185,13 @@ type LayerStore interface {
 	// underlying drivers can accept a "size" option.  At this time, most
 	// underlying drivers do not themselves distinguish between writeable
 	// and read-only layers.
-	Create(id, parent string, names []string, mountLabel string, options map[string]string, writeable bool) (*Layer, error)
+	Create(id string, parent *Layer, names []string, mountLabel string, options map[string]string, writeable bool, moreOptions *LayerOptions) (*Layer, error)
```

Comment:
Argument ordering nit: Wouldn't it make sense to have moreOptions immediately after options, instead of after writeable?

Reply:
Sure, why not?  Changed it.